xlators/mgmt/glusterd/src/glusterd-handshake.c

```diff
@@ -189,7 +187,6 @@ glusterd_get_client_per_brick_volfile(glusterd_volinfo_t *volinfo,
     glusterd_conf_t *priv = NULL;
     int32_t ret = -1;
 
-    GF_VALIDATE_OR_GOTO("glusterd", THIS, out);
     priv = THIS->private;
```

Comment:
nit: OK, glusterd_get_client_per_brick_volfile() should be a static function. It is only called from build_volfile_path() - and we don't need 'this' here, we only need priv, which we can pass to, right?
You can then remove the validation in the following line, as we know priv is not null already.

I wouldn't do it in this patch series which is already very big...

Reply:
I think the conversion of functions to static will itself be a separate patch as I've seen multiple instances of function not being static even if they should've been. 

And yes, there are places wherein only `private` is being used instead of any other component of `this`. I've added this point to the issue for further cleanup.